src/SongManager.cpp

```diff
@@ -2261,6 +2305,7 @@ class LunaSongManager: public Luna<SongManager>
 		ADD_METHOD( GetPopularSongs );
 		ADD_METHOD( GetPopularCourses );
 		ADD_METHOD( SongToPreferredSortSectionName );
+		ADD_METHOD( GetPreferredSortSongsBySectionName );

```

Comment:
Can we add lua docs for this (see how we've notated them ITGmania functions on the lua page)

e.g.
![image](https://github.com/itgmania/itgmania/assets/5017202/9fbf45f1-7f22-4f8b-bca6-785a42028fca)
****

Reply:
Gotcha